### DISCUSSION TITLE
How do I render a specific item from a collection?

### DISCUSSION THREAD

**Discussion by firedev** [bridgetownrb/bridgetown#810] [Q&A] [answered]
I want to add quotes grouped per author, so I make a collection:
```
quotes/author/quote.md
```

My idea was that I would have a `default.yml` with author details in each folder. But it seems that only flat lists are supported. I can see my quote in inspect though:

```
#<BridgetownCollection @label=quotes resources=[#<BridgetownResource::Base repo://quotes.collection/_quotes/author/quote.md>]>`
```

Is there a way to do something like `{% collections.quotes.author.quote %}`?

**Comment by jaredcwhite** [comment, marked as answer]
@firedev It sounds like you might want to look into resource relations:

https://www.bridgetownrb.com/docs/resources#resource-relations

That way your quotes can be related to specific authors in the authors collection, and you can lookup all quotes by certain authors, etc.

**Reply by firedev** [reply]
I see. That's more like it! Thank you!

**Comment by firedev** [comment]
I came back to quotes, trying to render a specific one. Is there a way to access it by slug, not just by number? 
```
{{ collections.quotes.resources[0] }}
```

**Comment by firedev** [comment]
Here is what I did
```
class Quote < Bridgetown::Component
  def initialize(slug:)
    @slug = slug
  end
end
```
```
{% for quote in @view_context.collections.quotes.resources %}
{% if quote.data.slug == @slug %}

> <b>{{ quote.content }}</b>
>
> — {{ @view_context.site.data.authors[quote.data.author].name }}

{% end %}
{% end %}
```
